src/opensearch_sdk_py/actions/internal/register_rest_actions_response_handler.py

```diff
@@ -18,14 +18,15 @@
 
 
 class RegisterRestActionsResponseHandler(ResponseHandler):
-    def __init__(self, next_handler: RequestResponseHandler) -> None:
+    def __init__(self, next_handler: RequestResponseHandler, request: OutboundMessageRequest) -> None:
```

Comment:
Introduce `OutboundResponseHandler`?

Reply:
> Introduce `OutboundResponseHandler`?

Not sure what you mean here.  Then again, this whole set of code is somewhat spaghetti so I'm not sure what I mean either.  I cobbled enough together to get it working and will try to figure out a better way to do all this.